### PR TITLE
enhancement: reference kyverno cli github action

### LINKED ISSUE #947: [Enhancement] Reference new CLI action in docs
### Description

The new CLI [installation Action](https://github.com/kyverno/action-install-cli) for GitHub makes using the CLI in GitHub Actions a cinch. Add this to the relevant docs.

### Slack discussion

_No response_

### PR DESCRIPTION
**Pull request by Chandan-DK**
## Related issue #
This PR adds a section for installation of Kyverno CLI using GitHub Actions
Closes #947 

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Review by chipzoller — CHANGES_REQUESTED** [review]
Please also add a section [here](https://kyverno.io/docs/testing-policies/#github-actions).

**Comment by Chandan-DK** [comment]
> Please also add a section [here](https://kyverno.io/docs/testing-policies/#github-actions).

The section includes a workflow yaml with a manual binary installation of the Kyverno CLI. Now that we have GitHub action for Kyverno CLI, we can remove the existing yaml and add a yaml which uses the action CLI. It'd look more elegant. What do you think?

**Comment by chipzoller** [comment]
Yes, it would be better to re-work that section to use the official Action.

**Comment by Chandan-DK** [comment]
> Yes, it would be better to re-work that section to use the official Action.

Sure. I'll do it

**Comment by Chandan-DK** [comment]
I have made the changes. You can take a look

File: content/en/docs/Testing Policies/_index.md [hunk 1]
```diff
@@ -50,44 +50,50 @@ The following is an example of a simple GitHub Actions workflow which may be use
...
-      - name: Download Kyverno CLI
-        run: |
-          curl -sLO https://github.com/kyverno/kyverno/releases/download/${{ env.VERSION }}/kyverno-cli_${{ env.VERSION }}_linux_x86_64.tar.gz
-          tar -xf kyverno-cli_${{ env.VERSION }}_linux_x86_64.tar.gz
-          ./kyverno version
+        uses: actions/checkout@v4
+      - name: Install Kyverno CLI
+        uses: kyverno/action-install-cli@v0.2.0
+        with:
+          release: 'v1.9.5'
```
Comment (chipzoller) [inline content/en/docs/Testing Policies/_index.md:74]:
```suggestion
          release: 'v1.11.0'
```

File: content/en/docs/Testing Policies/_index.md [hunk 2]
```diff
@@ -50,44 +50,50 @@ The following is an example of a simple GitHub Actions workflow which may be use
...
-![ci-fail](/images/ci-fail.png)
+![ci-fail][/images/ci-fail.png]
 
 Again, each step may be inspected for output. In the above screenshot, a Deployment manifest did not pass a validate policy and therefore caused the overall failure.
 
 For full usage details of the Kyverno CLI, refer to the [documentation](/docs/kyverno-cli/).
+
+
+[/images/ci-pass.png]: https://github.com/kyverno/kyverno/assets/103727912/54dbc4e0-ee9d-4ec5-953e-7e49ea6290af
+[/images/ci-fail.png]: https://github.com/kyverno/kyverno/assets/103727912/eba9ec26-0560-4971-92c7-ad2d30468550
```
Comment (chipzoller) [inline content/en/docs/Testing Policies/_index.md:99]: What is this here for? Please remove.

File: content/en/docs/Testing Policies/_index.md [hunk 3]
```diff
@@ -50,44 +50,50 @@ The following is an example of a simple GitHub Actions workflow which may be use
...
+        run: kyverno apply policies/ -r resources/
       - name: Test pre-defined cases
-        run: ./kyverno test tests/
+        run: kyverno test tests/
 ```
 
 Upon pull request to the repository containing this file, the GitHub Actions workflow will be triggered. If all tests succeed, the output will show a success for all steps.
 
-![ci-pass](/images/ci-pass.png)
+![ci-pass][/images/ci-pass.png]
```
Comment (chipzoller) [inline content/en/docs/Testing Policies/_index.md:85]: Not sure why you're changing the format of these links when they work as-is. This is standard Markdown.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Thanks for this. Little bit of cleanup.

File: content/en/docs/Testing Policies/_index.md (hunk 2 above)
Comment (Chandan-DK) [reply content/en/docs/Testing Policies/_index.md]: I used the image URLs here. Should have just modified the images in the folder.

**Comment by Chandan-DK** [comment]
> Thanks for this. Little bit of cleanup.

Done 👍

**Review by chipzoller — APPROVED** [review]
Thanks!

**Comment by Chandan-DK** [comment]
> Thanks!

Glad I could help!

**Pull request merged**